sources/packages/backend/apps/api/src/app.aest.module.ts

```diff
@@ -70,9 +70,10 @@ import { AuthModule } from "./auth/auth.module";
 import { ApplicationControllerService } from "./route-controllers/application/application.controller.service";
 import { InstitutionUserControllerService } from "./route-controllers/institution-user/institution-user.controller.service";
 import { WorkflowClientService, SequenceControlService } from "@sims/services";
+import { QueueRegistryModule } from "@sims/queue";
 
 @Module({
-  imports: [DatabaseModule, AuthModule],
+  imports: [DatabaseModule, AuthModule, QueueRegistryModule],
```

Comment:
I would recommend this module be declared as `Global` to avoid the need of consuming it on every single module. Again, just a recommendation, because seems to be something that every part of the application can have access to.